examples/metallic_systems.jl

```diff
@@ -46,8 +46,8 @@ scfres.energies
 # The fact that magnesium is a metal is confirmed
 # by plotting the density of states around the Fermi level.
 
-εs = range(minimum(minimum(scfres.eigenvalues)) - 1,
-           maximum(maximum(scfres.eigenvalues)) + 1, length=1000)
-Ds = DOS.(εs, Ref(basis), Ref(scfres.eigenvalues), temperature=4temperature)
```

Comment:
This was done to make the plots look nicer.

Reply:
They do look nicer (smoother), but also wronger (only very rough features are resolved). I think temperature is a good compromise?